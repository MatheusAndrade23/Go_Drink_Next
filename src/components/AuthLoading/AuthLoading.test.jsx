import { screen } from '@testing-library/react';
import { renderTheme } from '../../tests/render-base.js';
import { AuthLoading } from '.';

describe('<AuthLoading />', () => {
  it('should render', () => {
    const { container } = renderTheme(<AuthLoading />);
    expect(container).toMatchSnapshot();
  });
});
