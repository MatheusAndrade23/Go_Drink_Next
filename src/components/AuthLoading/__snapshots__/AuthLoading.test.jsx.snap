// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthLoading /> should render 1`] = `
.c0 {
  position: absolute;
  width: 100%;
  height: 100vh;
  top: 0;
  right: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: rgba(34,40,49,0.8);
  z-index: 300;
}

.c1 {
  width: 120px;
  height: 75px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c2 {
  width: 18px;
  height: 18px;
  border-radius: 50%;
  -webkit-animation: lfwXBT .5s alternate infinite;
  animation: lfwXBT .5s alternate infinite;
  background-color: #00ADB5;
}

.c2:nth-child(2) {
  -webkit-animation-delay: .16s;
  animation-delay: .16s;
}

.c2:nth-child(3) {
  -webkit-animation-delay: .32s;
  animation-delay: .32s;
}

.c3 {
  font-size: 22px;
  margin: auto;
  color: #00ADB5;
  font-weight: bold;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      />
      <div
        class="c2"
      />
      <div
        class="c2"
      />
      <span
        class="c3"
      >
        loading
      </span>
    </div>
  </div>
</div>
`;
